scripts/pythonscripts/sync_postgres_to_mssql.py

```diff
@@ -0,0 +1,228 @@
+import psycopg2
```

Comment:
For formatting, can we run `black` and `isort` on this file? These are python packages to install.